src/main/webapp/js/omniComment.js

```diff
@@ -224,13 +224,9 @@ $(document).ready(function() {
             studentCommentPanelBody.show();
         } else { // to show feedback question + feedback session panel
             var commentListRegionForFeedbackResponse = commentToShow.closest('tr');
-            // a fbResponse in instructorCommentsPage consists of the 4 table rows from the
-            // comment list region row to the giver recipient row
-            // The following statement shows the rows between comment list region row and giver recipient row
-            // including the comment list region row and giver recipient row.
-            // See stackoverflow post: http://stackoverflow.com/questions/2770588/jquery-prevuntil-include-start-selector-and-end-selecter
-            commentListRegionForFeedbackResponse.prevUntil('.feedback-response-giver-recipient-row')
-                    .addBack().prev('.feedback-response-giver-recipient-row').addBack().show();
+            var rowsToShowClassName = commentListRegionForFeedbackResponse.attr('class');
+            $('.' + rowsToShowClassName).show();
```

Comment:
Unrelated to this, in ES6 we can do string templating which will make this more obvious, maybe one day we might be able to make TEAMMATES ES6 ready :)